collateral-info.yaml

```diff
@@ -285,6 +292,38 @@ collateral-defs:
     collateral-symbol: RWA009
     collateral-info-url: https://manual.makerdao.com/collateral/real-world-assets-in-detail/hvbank
 
+  RWA010: &RWA010
+    <<: *CFG-DROP
+    collateral-desc: Revolving Credit Facility formed of the following collateral types: Senior Secured Facilities, Forward-Flow Arrangements, Whole loans or receivables purchases, Investment Grade Structured Credit Products (ABS bonds).
+    collateral-longname: Centrifuge Blocktower Credit Series 1 DROP Token
+    collateral-shortname: BlockTower Series 1 DROP
```

Comment:
Maybe Series 1 -> S1? Feels like it might be pushing characters for a short-name as is.

Reply:
agree.